newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageHeader.js

```diff
@@ -52,22 +52,23 @@ export const HomePageHeader = ({
         >
           <Column noMargin>
             <Line noMargin>
-              <IconButton
-                size="small"
-                id="main-toolbar-project-manager-button"
-                onClick={onOpenProjectManager}
-                tooltip={t`Project Manager`}
-                color="default"
-                disabled={!hasProject}
-              >
-                <ProjectManagerIcon />
-              </IconButton>
               {!!hasProject && (
-                <SaveProjectIcon
-                  id="main-toolbar-save-button"
-                  onSave={onSave}
-                  canSave={canSave}
-                />
+                <>
+                  <IconButton
+                    size="small"
+                    id="main-toolbar-history-button"
+                    onClick={onOpenVersionHistory}
+                    tooltip={t`Open version history`}
+                    color="default"
+                  >
+                    <HistoryIcon />
```

Comment:
added history button on the bar of the homepage to be consistent with the project bar, and avoid icons jumping